ggml/include/ggml-backend.h

```diff
@@ -236,6 +277,27 @@ extern "C" {
     GGML_API void ggml_backend_view_init(struct ggml_tensor * tensor);
 
 
+    //
+    // CPU backend
+    //
+
+    GGML_API ggml_backend_t ggml_backend_cpu_init(void);
+
+    GGML_API bool ggml_backend_is_cpu                (ggml_backend_t backend);
```

Comment:
I don't intend to change these functions at the moment. Most of the functions that need these checks, like `ggml_backend_cpu_set_n_threads`, operate on a `ggml_backend_t` object, so it is still convenient to have a function to check if a `ggml_backend_t` belongs to a specific backend. After all the backends have been adapted to the new interface this could be re-evaluated.